frontend/src/components/modals/removeAvatar/RemoveAvatarModal.tsx

```diff
@@ -0,0 +1,40 @@
+import { IModal } from "../modals.interfaces";
+import { useRef } from "react";
+import { useOnClickOutside } from "usehooks-ts";
+import useRemoveAvatar from "./useRemoveAvatar";
+import { ModalSpan } from "../GenericModalComponents.styled";
+import { PrimaryButton, SecondaryButton } from "../../buttons/Button.styled";
+import {
+    RemoveAvatarButtonsWrapper,
+    RemoveAvatarWrapper
+} from "./RemoveAvatarModal.styled";
+import { ErrorIndicator } from "assets/styles/theme";
+
+const RemoveAvatarModal = ({ hideModal }: IModal) => {
+    const { removeAvatarHandler, error } = useRemoveAvatar(hideModal);
+
+    const ref = useRef(null);
```

Comment:
U pick wrong thing, on rebase.

Use this:

```suggestion
   const { ref } = useClickOutside(hideModal);
```